fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -204,24 +211,24 @@ ACTOR Future<Void> trackShardBytes(
 					ASSERT( false );
 				}
 				// handle read bandkwith status
-				if (newReadBandwidthStatus != readBandwidthStatus) {
```

Comment:
trackShardBytes()'s actor name is not longer reflecting its functionality. Would you mind opening another PR to change the name to "trackShardMetrics()". In the caller, it also needs to change shardSize to shardMetrics.

Reply:
OK